### PR TITLE
Addition of Episode 15.2 : Infinite Space monsters

### PR DESCRIPTION
**Pull request by hannicaldummy**
* **Addressed Issue(s)**: https://github.com/rathena/rathena/issues/6448

* **Server Mode**: Renewal

* **Description of Pull Request**: All major mobs information has been taken from divine-pride and basic stats from original mobs version.

A Aggressive mode added as we can see in https://www.youtube.com/watch?v=d4RDE4ty-VM

### CODE REVIEW

**Comment by mundussan** [comment]
I think the YAML structure or file encoding is wrong. Did you done it by hand using a text editor or you used a program to automatize that?

**Comment by hannicaldummy** [comment]
By hand using Notepad++, but there is no errors in map-server and the monsters shows normally with @monster command ingame

**Comment by aleos89** [comment]
You also have a few mobs with over 10 drops.

**Comment by hannicaldummy** [comment]
> You also have a few mobs with over 10 drops.

I checked here, al mobs have 6 to 8 drops, only MVPs have 8 normal drops and 3 mvp drops, does MVP drops count into these 10 normal limit?

File: db/re/mob_db.yml [hunk 1]
```diff
@@ -82968,120 +82968,2610 @@ Body:
...
+      - Item: Wing_Of_Fly
+        Rate: 500
+      - Item: Main_Gauche
+        Rate: 20
+      - Item: Apple
+        Rate: 125
+      - Item: Poporing_Card
+        Rate: 10
+        StealProtected: true
+  - Id: 1056
```
Comment (aleos89) [inline db/re/mob_db.yml:83360]: You're re-defining Smokie here? This is why it's saying the mob drop is over 10.

Comment (hannicaldummy) [reply db/re/mob_db.yml:83360]: It was a typo, forgot to change his ID and aegisname, done :D

File: db/re/mob_db.yml [hunk 2]
```diff
@@ -82968,120 +82968,2610 @@ Body:
...
+      - Item: Picky_Card
+        Rate: 20
+        StealProtected: true
+  - Id: 3388
+    AegisName: MIN_CONDOR
+    Name: Condor
+    Level: 101
+    Hp: 40000
+    BaseExp: 1350
+    JobExp: 1300
```
Comment (Atemo) [inline db/re/mob_db.yml:83171]: Inverted
```suggestion
    BaseExp: 1300
    JobExp: 1350
```

File: db/re/mob_db.yml [hunk 3]
```diff
@@ -82968,120 +82968,2610 @@ Body:
...
+    Attack: 701
+    Attack2: 945
+    Defense: 7
+    MagicDefense: 5
+    Str: 14
+    Agi: 7
+    Vit: 6
+    Dex: 13
+    Luk: 5
```
Comment (Atemo) [inline db/re/mob_db.yml:83180]:
```suggestion
    Defense: 150
    MagicDefense: 5
    Str: 112
    Agi: 53
    Vit: 72
    Int: 38
    Dex: 123
    Luk: 28
```

File: db/re/mob_db.yml [hunk 4]
```diff
@@ -82968,120 +82968,2610 @@ Body:
...
+      - Item: Dryad_Card
+        Rate: 1
+        StealProtected: true
+  - Id: 3421
+    AegisName: MIN_TOAD
+    Name: Infinite Toad
+    Level: 105
+    Hp: 900000
+    BaseExp: 13800
+    JobExp: 17250
```
Comment (Atemo) [inline db/re/mob_db.yml:84772]:
```suggestion
    BaseExp: 12000
    JobExp: 15000
```

File: db/re/mob_db.yml [hunk 5]
```diff
@@ -82968,120 +82968,2610 @@ Body:
...
+      - Item: Min_Toad_Card
+        Rate: 1
+        StealProtected: true
+  - Id: 3422
+    AegisName: MIN_VAGABOND_WOLF
+    Name: Infinite Vagabond Wolf
+    Level: 105
+    Hp: 900000
+    BaseExp: 13800
+    JobExp: 17250
```
Comment (Atemo) [inline db/re/mob_db.yml:84822]:
```suggestion
    BaseExp: 12000
    JobExp: 15000
```

File: db/re/mob_db.yml [hunk 6]
```diff
@@ -82968,120 +82968,2610 @@ Body:
...
+      - Item: Min_Vagabond_Wolf_Card
+        Rate: 1
+        StealProtected: true
+  - Id: 3423
+    AegisName: MIN_VOCAL
+    Name: Infinite Vocal
+    Level: 105
+    Hp: 900000
+    BaseExp: 13800
+    JobExp: 17250
```
Comment (Atemo) [inline db/re/mob_db.yml:84872]:
```suggestion
    BaseExp: 12000
    JobExp: 15000
```

File: db/re/mob_db.yml [hunk 7]
```diff
@@ -82968,120 +82968,2610 @@ Body:
...
+      - Item: Min_Orc_Hero_Card
+        Rate: 1
+        StealProtected: true
+  - Id: 3430
+    AegisName: MIN_TAO_GUNKA
+    Name: Infinite Tao Gunka
+    Level: 109
+    Hp: 8550000
+    BaseExp: 100000
+    JobExp: 100000
```
Comment (Atemo) [inline db/re/mob_db.yml:85257]:
```suggestion
    BaseExp: 200000
    JobExp: 200000
```

File: db/re/mob_db.yml [hunk 8]
```diff
@@ -82968,120 +82968,2610 @@ Body:
...
-#    AegisName: MIN_G_MEGALITH
+  - Id: 3384
+    AegisName: MIN_PORING
+    Name: Poring
+    Level: 100
+    Hp: 25000
+    BaseExp: 1200
+    JobExp: 1200
+    Attack: 843
+    Attack2: 1160
```
Comment (Atemo) [inline db/re/mob_db.yml:82979]: Capture d’écran (612)
The values displayed on DP are the final values, we need the "raw" values for Attack and Attack2.
You can find the raw values on DP API, here it should be `Attack: 794` and `Attack2: 67` for MIN_PORING

Capture d’écran (611)
The Attack/Attack2 need to be corrected for all monsters

**Review by Atemo — CHANGES_REQUESTED** [review]

File: db/re/mob_db.yml (hunk 8 above)
Comment (hannicaldummy) [reply db/re/mob_db.yml:82979]: How can I check the DP API?

Comment (mundussan) [reply db/re/mob_db.yml:82979]: > How can I check the DP API?

https://www.divine-pride.net/api
here

**Comment by Everade** [comment]
@Atemo i sent a patch to @Lemongrass3110 which should have adressed it all.

**Review by Atemo — APPROVED** [review]
Good for me!

**Pull request merged**